tests/unit-core/test-api-set-and-clear-error-flag.c

```diff
@@ -16,6 +16,16 @@
 #include "jerryscript.h"
 #include "test-common.h"
 
+static void
+compare_str (jerry_value_t value, const char *str_p, size_t str_len)
+{
+  jerry_size_t size = jerry_get_string_size (value);
+  JERRY_ASSERT (str_len == size);
+  jerry_char_t str_buff[size];
```

Comment:
I just remember that there is a JERRY_VLA macro for defining such arrays to allow windows compatibility.

Reply:
Sure, I changed it to ```JERRY_VLA``` and updated the PR.